dist_q_learning/main.py

```diff
@@ -207,7 +213,7 @@ def run_main(cmd_args):
             num_actions=env.num_actions,
             env=env,
             gamma=0.99,
-            lr=1.,
```

Comment:
Todo - We probably need to add a case distinction here to have lr=.1 for GLN/continuous, and 1 if we use a deterministic finite environment.